tests/smoke-test-general.bats

```diff
@@ -350,4 +350,35 @@ services:
 	[[ $(echo "$output" | grep -c "feaTures.Alpha-beta_zulu.docksal") -eq 0 ]]
 	[[ "${output}" =~ "The VIRTUAL_HOST has been modified from feaTures.Alpha-beta_zulu.docksal to features.alpha-beta-zulu.docksal to comply with browser standards." ]]
 	unset output
-}
\ No newline at end of file
+}
+
+@test "fin share" {
+	#[[ $SKIP == 1 ]] && skip
+
+        # Send all mail to /bin/true
+        echo "sendmail_path=/bin/true" >> .docksal/etc/php/php.ini
+        # Initialize the Project
+        fin init
```

Comment:
Let's move this up, below `fin init` or `fin restart tests`, this way the extra init won't be necessary. Is does introduce a dependency on the previous test though, but we do have other tests happening linearly now, which does save time vs starting every test from scratch.

Reply:
@lmakarov I don't think we should introduce a dependency at all which is why I'm suggesting the `fin init` here to completely start from scratch on the project. That way regardless where in the line we run it. We know it will work.